.circleci/config.yml

```diff
@@ -18,21 +18,21 @@ jobs:
           name: 'Pylinting'
           command: |
             . venv/bin/activate
-            pylint target_snowflake -d C,W,unexpected-keyword-arg,duplicate-code
+            pylint target_snowflake
 
       - run:
           name: 'Unit Tests'
           command: |
             . venv/bin/activate
             export LOGGING_CONF_FILE=$(pwd)/sample_logging.conf
-            nosetests --where=tests/unit
+            pytest tests/unit
```

Comment:
do we know what's current coverage %? you can check by installing `pytest-cov` plugin.
Might also be a good idea to set current % as the mix so that any new future code would need to be covered too to pass the tests.

Reply:
i just wanted to add parquet support :)

Reply:
I will create jira

Reply:
great idea, but probably not part of refactoring exercise

Reply:
ok, added in 107b947e8112448c020dfd98bc42d184718ed105

integration test current test coverage % will be added as part of another commit.

Reply:
sorry :( ! it was just a suggestion, you don't have to do it.

Reply:
integration test coverage current percent added by 72d3f5884253076a2eabc9299f892fe11bc7d374

Current test coverage:
Unit tests: 52%
Integration tests: 86%

Reply:
> sorry :( ! it was just a suggestion, you don't have to do it.

too late :)

Reply:
😅 , going forward, please do not take every comment I make on PR as a request to change, sometime, I'm just curious so I ask.

Reply:
no, it's all fine, keep doing that please. But I'll add them as separated PRs, once this big one got merged.